src/components/ObsDetails/ActivityItem.js

```diff
@@ -63,6 +66,19 @@ const ActivityItem = ( {
     }
   );
 
+  const showNoInternetIcon = ( ) => (
+    <View className="mr-3">
+      <IconMaterial name="network-check" size={30} />
```

Comment:
I have not seen this icon used in the context of showing that no internet connection is available. Standard would be `wifi-off`.
Can you add an i18n'ed `accessibilityLabel` to this new component. That would read to the user what is not available.

Both comments also applicable in the other places of this icon use.